public/locales/auset-fr.json

```diff
@@ -0,0 +1,426 @@
+      "setupTitle": "Configurer le Vagabond",
+      "setup": "<i>1.</i> Placez votre <strong>pion</strong> dans n'importe quelle forêt.<br/><i>2.</i> Mélangez le paquet de quêtes. Piochez 3 <strong>quêtes</strong> à distribuer à proximité.<br/><i>3.</i> Placez le <Sac/>, la <Boot/>, le <Hammer/> et l'<Sword/> <strong>objets de destruction</strong > <i>(marqué d'un\"R\")</i> sous les ruines au hasard, sauf si cela a déjà été fait.<br/><i>4.</i> Mettez le\"{{vagabond}}\" <strong>carte de personnage</strong> dans votre emplacement de carte de personnage. Placez les <InitialStartingItems>, </InitialStartingItems> et <FinalStartingItem/> <strong>éléments de départ</strong> <i>(marqués d'un\"S\")</i> dans votre Cartable et sur leurs pistes correspondantes, si nécessaire.",
+      "setup_vagabondSetUp": "<i>1.</i> Placez votre <strong>pion</strong> dans n'importe quelle forêt.<br/><i>2.</i> Mélangez le paquet de quêtes. Piochez 3 <strong>quêtes</strong> à distribuer à proximité.<br/><i>3.</i> Mettez les objets supplémentaires <Bag/>, <Boot/>, <Hammer/> et <Sword/> <strong>détruire</strong> <i>(marqué d'un\"R\")</i> sous les ruines au hasard <i>(avec les objets existants)</i>.<br/><i>4.</i> Mettez la\"{{vagabond}}\" <strong>carte de personnage</strong> dans votre emplacement de carte de personnage. Mettez les <InitialStartingItems>, </InitialStartingItems> et <FinalStartingItem/> <strong>éléments de départ</strong> <i>(marqué d'un\"S\")</i> dans votre sacoche et sur leurs pistes correspondantes, si nécessaire."
+    },
+    "warlord": {
+      "name": "Seigneur des centaines",
+      "summaryTitle": "Jouer le Seigneur des centaines",
+      "summary": "<p>En tant que Seigneur des Centaines, vous marquez des points lorsque vous <strong>opprimez</strong> vos ennemis. À la fin de votre tour, plus vous dirigez de clairières qui n'ont <i>aucun</i> ennemi pièces - pas de guerriers, pas de bâtiments, rien - plus vous marquez de points.</p><p>Pour gagner en puissance et attirer des guerriers, vous devez gagner des objets et les ajouter à votre imposant <strong>Hoard</strong>. Les bottes, les sacs et les pièces augmentent votre <strong>Commandement</strong>, tandis que les marteaux, le thé, les épées et l'arbalète augmentent votre <strong>Prouesse</strong>. Votre <strong>Mépris du Commerce</strong> signifie vous marquez rarement des points en fabriquant des objets, mais vous pouvez voler des objets à vos ennemis avec vos <strong>pilleurs</strong> !</p><p>À la tête des centaines se trouve votre seigneur de guerre, un guerrier-démagogue dont l'humeur <strong>inconstante< /strong> vous donne une capacité pour le tour. Votre seigneur de guerre est obsédé par la thésaurisation, donc plus vous gagnez d'objets, moins vous aurez le choix.</p><p>Déclarez-vous être la vraie voix du Woodland, vous pouvez inciter <strong>mo bs</strong>, qui détruisent les bâtiments et les jetons ennemis, et attaquent les ruines pour leurs objets.</p>",
```

Comment:
Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.